apps/scan/frontend/src/api.ts

```diff
@@ -16,6 +16,8 @@ import {
   createUiStringsApi,
 } from '@votingworks/ui';
 
+const PRINTER_STATUS_POLLING_INTERVAL_MS = 100;
```

Comment:
Didn't want to put this in `libs/printing` because not sure whether we'll want the same interval for all apps.